src/modules/generateRandomNumber.js

```diff
@@ -8,6 +8,19 @@
  */
 function generateRandomNumber() {
   /* Write your code here */
+  const availableNums = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const result = [];
+
+  const getRandomN = () => Math.floor(Math.random() * availableNums.length);
+
+  result.push(availableNums.splice(getRandomN(), 1)[0]);
```

Comment:
The use of `splice` method here can be confusing. It's better to avoid changing the original array. Consider using a different approach to select a random element from the array without modifying it.